packages/sdk-core/src/model/http/model/submit/request/BacktraceSubmitSourceCode.ts

```diff
@@ -0,0 +1,61 @@
+export interface TextBacktraceSubmitSourceCode {
```

Comment:
How about `BacktraceSubmitSourceCodeData` instead? In the UI source code is used for multiple purposes. We can check it with the UI team, but I think more generic name makes sense

Reply:
renamed to `BacktraceSubmitTextSourceCode`, I think the `text` part should stay, as it is describing what is actually in the model